### PR TITLE
add more log for auth when failed

### PR DESCRIPTION
**Pull request by lei-flex**
add more log when login failed.

### CODE REVIEW

**Comment by tylerflex** [comment]
@lei-flex just a heads up, for the front end tests to pass, need to 
```py
black . --line-length 100
```
to fix any formatting issues and then
```py
python lint.py
```
to run linter to ensure it's a clean lint / (10/10 score)

I just went ahead and fixed the issues but just a note for next time. Thanks!

File: tidy3d/web/auth.py [hunk 1]
```diff
@@ -128,4 +123,41 @@ def get_credentials() -> None:
...
+
+def __get_credential_from_stored_file():
+    with open(credential_path, "r", encoding="utf-8") as fp:
+        auth_json = json.load(fp)
+    email = auth_json["email"]
+    password = auth_json["password"]
+    return email, password
+
+
+def __get_credential_from_env():
```
Comment (tylerflex) [inline tidy3d/web/auth.py:137]: Could you add docstrings to these functions? just to describe what the logic is doing. For example:
```py
def _get_credential_from_env():
    """Try to get email and password from environment variables, if not encoded, encode it."""
```
or something similar, just to help us know what these functions should do, thanks

**Review by tylerflex — CHANGES_REQUESTED** [review]

**Comment by momchil-flex** [comment]
I did various tests, everything looks good. My only recommendation would be to bring back the `TIDY3D_ENV` variable as I mentioned on slack.

**Comment by lei-flex** [comment]
> we should have the another ticket for TIDY3D_WEB_ENV. 
the purpose for this ticket is just to show more detail message during log and keep other logic no chang.e

**Comment by momchil-flex** [comment]
Then fine by me to merge this.

**Review by tylerflex — APPROVED** [review]

**Pull request merged**